src/controller/data_model/controller-clusters.matter

```diff
@@ -2934,6 +2934,26 @@ provisional client cluster OvenMode = 73 {
   command ChangeToMode(ChangeToModeRequest): ChangeToModeResponse = 0;
 }
 
+/** This cluster supports remotely monitoring and controling the different typs of
+            functionality available to a drying device, such as a laundry dryer. */
+client cluster LaundryDryerControls = 74 {
+  enum DrynessLevelEnum : enum8 {
+    kLow = 0;
+    kNormal = 1;
+    kExtra = 2;
+    kMax = 3;
+  }
+
+  readonly attribute optional DrynessLevelEnum supportedDrynessLevels[] = 0;
```

Comment:
I am looking to compare with spec in 7766. In there I do not seem to find optional conformance for either this or selectedDrynessLevel

Reply:
I have updated the xml with optional=false.